agent/command/results_xunit.go

```diff
@@ -168,9 +175,8 @@ func (c *xunitResults) parseAndUploadResults(ctx context.Context, conf *internal
 			cumulative = addTestCasesForSuite(suite, idx, conf, cumulative)
 		}
 	}
-
-	if len(cumulative.tests) == 0 {
-		return errors.New("no test results found")
+	if len(reportFilePaths) == numInvalid {
+		return errors.New("all given file paths are invalid")
```

Comment:
I imagine users wouldn't understand tihs. Can this specifically state that it's because the matching files were either directories or nonexistent?

Reply:
I'm not too concerned here since we log if it's one of the above problems